stdlib/3/os/__init__.pyi

```diff
@@ -3,7 +3,7 @@
 
 # based on http: //docs.python.org/3.2/library/os.html
 
-from builtins import OSError as error
+from builtins import function, OSError as error
```

Comment:
Sure thing! And strange, because I tested the initial version with the latest PyPI installed mypy, pointing its typeshed to this branch and it worked fine (I also tested the `Callable[..., Any]` version and it works as expected as well). The test file I used was the following:

```python
import os

assert os.symlink in os.supports_dir_fd
assert os.chdir in os.supports_fd
assert os.access in os.supports_effective_ids
assert os.stat in os.supports_follow_symlinks
```